2016-09/september-14.md

```diff
@@ -0,0 +1,27 @@
+## September 14
+
+### Attendees
+
+* [Julen](https://github.com/julen) (Evernote)
+* [Leandro](https://github.com/unho) (Translate House)
+* [Ryan](https://github.com/phlax) (Translate House)
+* [Taras](https://github.com/ta2-1) (Translate House)
+
+
+### Ongoing work
+
+* Migrations: there are some integrity errors when upgrading from 2.5 and
+  switching from MyISAM to InnoDB
+  ([#5246](https://github.com/translate/pootle/issues/5246))
+* Work on due dates has started
+  ([feature/duedats](https://github.com/translate/pootle/tree/feature/duedates))
```

Comment:
- Work on vfolders continues and will be landed soon so it unblocks stats and monolingual PRs.